### DISCUSSION TITLE
Version 1.3.0

### DISCUSSION THREAD

**Discussion by Birdulon** [Melledy/Grasscutter#1740] [Announcements]
## What's Changed
* Update ru-RU.json by @TotalyNotOndre in https://github.com/Grasscutters/Grasscutter/pull/1683
* update zh-CN.json by @2y8e9h22 in https://github.com/Grasscutters/Grasscutter/pull/1684
* More literate README_ru-RU.md by @IMrProYTI in https://github.com/Grasscutters/Grasscutter/pull/1691
* Update README.md by @AwayJob in https://github.com/Grasscutters/Grasscutter/pull/1695
* Fix QuestEncryptionKeys Path by @omg-xtao in https://github.com/Grasscutters/Grasscutter/pull/1696
* 3.0 Support (Backrooms Edition)  by @4Benj in https://github.com/Grasscutters/Grasscutter/pull/1698
* Update README_zh-CN.md by @MlgmXyysd in https://github.com/Grasscutters/Grasscutter/pull/1708
* partial fix of dungeon by @pfyy in https://github.com/Grasscutters/Grasscutter/pull/1706
* Dutch language updates For README by @Pikachubolk in https://github.com/Grasscutters/Grasscutter/pull/1709
* fix dungeon by @pfyy in https://github.com/Grasscutters/Grasscutter/pull/1710
* Update translation by @Tesutarin in https://github.com/Grasscutters/Grasscutter/pull/1711
* Jp support by @Kei-Luna in https://github.com/Grasscutters/Grasscutter/pull/1712
* Fixing Translation errors from previous commit by @Pikachubolk in https://github.com/Grasscutters/Grasscutter/pull/1713
* Add permission clear and list commands by @jie65535 in https://github.com/Grasscutters/Grasscutter/pull/1715
* Update README.md Sequel by @AwayJob in https://github.com/Grasscutters/Grasscutter/pull/1717
* fix: avatar energy transform in proudskill by @phuchptty in https://github.com/Grasscutters/Grasscutter/pull/1728
* fix serenitea pot by @pfyy in https://github.com/Grasscutters/Grasscutter/pull/1726
* Update default Banners.json by @loulou310 in https://github.com/Grasscutters/Grasscutter/pull/1719
* Console output player command by @Kawaa-qwq in https://github.com/Grasscutters/Grasscutter/pull/1722
* Custom Teams by @GanyusLeftHorn in https://github.com/Grasscutters/Grasscutter/pull/1731
* Replace Java-Express with Javalin by @4Benj in https://github.com/Grasscutters/Grasscutter/pull/1733
* fix serenitea pot main building by @pfyy in https://github.com/Grasscutters/Grasscutter/pull/1737
* Update ru-RU.json by @TotalyNotOndre in https://github.com/Grasscutters/Grasscutter/pull/1738
* Plugin Manager localisation by @TotalyNotOndre in https://github.com/Grasscutters/Grasscutter/pull/1739

## New Contributors
* @IMrProYTI made their first contribution in https://github.com/Grasscutters/Grasscutter/pull/1691
* @AwayJob made their first contribution in https://github.com/Grasscutters/Grasscutter/pull/1695
* @pfyy made their first contribution in https://github.com/Grasscutters/Grasscutter/pull/1706
* @Pikachubolk made their first contribution in https://github.com/Grasscutters/Grasscutter/pull/1709
* @Kei-Luna made their first contribution in https://github.com/Grasscutters/Grasscutter/pull/1712
* @phuchptty made their first contribution in https://github.com/Grasscutters/Grasscutter/pull/1728

**Full Changelog**: https://github.com/Grasscutters/Grasscutter/commits/v1.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Grasscutters/Grasscutter/releases/tag/v1.3.0'>Version 1.3.0</a>.</em>

**Comment by wajunye** [comment]
greatest work! 
appreciate.

**Comment by ChisatoNishikigi73** [comment]
appreciate.
